### PR TITLE
Add support for ivy/swiper

### PR DESCRIPTION
**Pull request by bricewge**

### CODE REVIEW

**Comment by bbatsov** [comment]
If the first commit messag - `to` should be `for`.

**Comment by bricewge** [comment]
The swiper faces are not the best but at least it stick it's not color pasta anymore.

The typo in fixed now.

File: solarized.el [hunk 1]
```diff
@@ -1148,6 +1148,15 @@ customize the resulting theme."
        ((,class (:foreground ,magenta :background ,base02))))
      `(info-user-option-ref-item
        ((,class (:foreground ,orange :background ,base02))))
+  ;;;;; ivy
```
Comment (bbatsov) [inline solarized.el]: This line is not properly indented.

**Comment by bricewge** [comment]
The indentation is fixed now.

**Comment by bbatsov** [comment]
:+1:

**Pull request merged**